packages/core/src/routes/sign-in-experience.ts

```diff
@@ -67,6 +68,15 @@ export default function signInExperiencesRoutes<T extends AuthedRouter>(router:
         validateSignUp(signUp, enabledConnectors);
       }
 
+      if (signIn) {
+        if (signUp) {
+          validateSignIn(signIn, signUp, enabledConnectors);
+        } else {
+          const signInExperience = await findDefaultSignInExperience();
+          validateSignIn(signIn, signInExperience.signUp, enabledConnectors);
```

Comment:
nit: maybe this?
```ts
const signUp = signUp ?? ((await findDefaultSignInExperience()).signUp);
```

Reply:
if (signIn && signUp)

else if (signIn)

Reply:
@charIeszhao it's not allowd to do that.

Reply:
updated @simeng-li